src/main.cpp

```diff
@@ -74,21 +79,172 @@ void micromouse_main()
   LOG_INIT();
 
   Orientation::getInstance().resetHeading();
-
+    
   MenuItem items[] = {
-    { "RUN", run },
-    { "KAOS", kaos },
-    { "TURN", turn },
-    { "CHK", check },
-    { "OPT", options },
-    {}
-  };
+        { "AUTO", autoMode },
+	    { "CLR", clear },
+	    { "MZCLR", mazeClear },
+	    {}
+    };
+	/*MenuItem items[] = {
+		{ "RUN", run },
+		{ "KAOS", kaos },
+		{ "TURN", turn },
+		{ "CHK", check },
+		{ "OPT", options },
+		{}
+	  };*/
 
   Menu menu(items, false);
 
   for (;;) menu.run();
 }
 
+void autoMode()
+{
+	//MUST CLEAR MANUALLY AT VERY BEGINNING
+	PersistantStorage::setTargetXLocation(APAC_TARGET_X);
+	PersistantStorage::setTargetYLocation(APAC_TARGET_Y);
```

Comment:
I would argue that we can remove this. We can upload old code to set any target location we want, and then upload the auto mode code which won't allow us to change the location anymore. In my eyes that's a good enough solution and gives us maximum flexibility for testing.